Tensile/SolutionStructs.py

```diff
@@ -2705,6 +2705,9 @@ def assignDerivedParameters(state):
     if state["WavefrontSize"] == 32 and state["KernelLanguage"] == "Source":
       reject(state, "WavefrontSize=32 not yet supported for source kernels.")
 
+    if state["MinKForGSU"] not in [16,32,64,128,256]:
```

Comment:
This is not what I said.
We do not need out of range check.
It is already implemented for all parameters.

What I asked is to reject GSU=1 and MiKForGSU!=default value.
We already have similar check for GSU=1 and MultiBuffer.
Please put the new check after other checks for GSU=1.